tests/tests_data/datasets/test_env.py

```diff
@@ -109,6 +110,7 @@ def env_auto_test(fabric: Fabric, num_workers):
         pass
 
 
+@RunIf(skip_windows=True)
```

Comment:
sorry, my suggestion won't work because RunIf is not importable from this test package, as it is defined in fabric only.

So this one will have to do:

```py
@pytest.mark.skipif(sys.platform() == "win32", reason="multiprocessing issues on windows")
```

Reply:
Test passed, so all looks good after all. Do you prefer to change it this way anyway?

Reply:
no preference, i thought we had this limitation but apparently not